scripts/popup.js

```diff
@@ -300,6 +300,27 @@ function settings() {
   $('#setting-page').show()
 }
 
+function share(){
+  const webstoreURL = 'https://chrome.google.com/webstore/detail/wayback-machine/fpnmgdkabkmnadcjpehmlllkndpkmiak'
+  function showToast(toastMsg){
+    toastMsg.classList.add('show')
+    setTimeout(function(){
+      toastMsg.classList.remove('show')
+    }, 1000)
+  }
+  navigator.clipboard.writeText(webstoreURL).then(function() {
+    const toastMsg = document.getElementsByClassName('toast')
```

Comment:
Move this line out side of this scope and declare it there once and remove line no-307

Reply:
Moved it, but still confused about line 307. Should I remove the timeout function?